### PR TITLE
delete unclassified

### PR DESCRIPTION
**Pull request by gmalinve**

### CODE REVIEW

**Comment by gmalinve** [comment]
@Samuelopez-ansys @maxcapodi78 
Why do we undo the design if the result of the intersection is an unclassified object?
It's in the delete() method in Primitives.py: 
`unclassified1 = list(self.oeditor.GetObjectsInGroup("Unclassified"))
if unclassified != unclassified1:
    self._odesign.Undo()
    self.logger.error("Error in intersection. Reverting Operation")
    return`
it doesn't necessarily means that the intersect method failed.
In fact in test _test_26_unclassified_object_ we use the native API to be able to get the unclassified objects.

File: _unittest/test_07_Object3D.py [hunk 1]
```diff
@@ -537,6 +537,11 @@ def test_26_unclassified_object(self):
         self.aedtapp.modeler.oeditor.Intersect(vArg1, vArg2)
         assert box1 in self.aedtapp.modeler.unclassified_objects
 
+    def test_26a_delete_unclassified_object(self):
+        unclassified = self.aedtapp.modeler.unclassified_objects
+        assert self.aedtapp.modeler.delete(unclassified)
+        assert len(self.aedtapp.modeler.unclassified_objects) != unclassified
```
Comment (SMoraisAnsys) [inline _unittest/test_07_Object3D.py:543]: Shouldn't you test that the value is equal to 0 since the got deleted ?

Comment (gmalinve) [reply _unittest/test_07_Object3D.py:543]: @SMoraisAnsys yes I could but it's the same at the end. I can add a new line.

**Comment by Samuelopez-ansys** [comment]
@gmalinve I think if the number of unclassified objects is different before the operation and after, it means the operation failed, right?

**Review by Samuelopez-ansys — APPROVED** [review]

**Comment by gmalinve** [comment]
> @gmalinve I think if the number of unclassified objects is different before the operation and after, it means the operation failed, right?

@Samuelopez-ansys does the intersection fail if you have an unclassified object as a result?

**Comment by Samuelopez-ansys** [comment]
@gmalinve Unclassified object is not desired. It means the boolean operation failed and it was not correct. This extra step, is just avoiding the user to Undo later, but the method is returning False in this case, so for me it is expected behavior.

**Pull request merged**